### PR TITLE
[W6.5][W10-B4]Joshua Lee Kai Sheng

### PR DESCRIPTION
**Pull request by jlks96**
-added application icon + edited font

### CODE REVIEW

File: src/seedu/addressbook/Main.java [hunk 1]
```diff
@@ -20,8 +22,10 @@
 
     @Override
     public void start(Stage primaryStage) throws Exception{
+
```
Comment (tshradheya) [inline src/seedu/addressbook/Main.java:25]: Avoid unnecessary changes :P

**Review by tshradheya — COMMENTED** [review]
Good job learning JavaFx :)

**Comment by jlks96** [comment]
Thanks. Acknowledged and closed.
